payments/swish/seamless-view.md

```diff
@@ -18,13 +18,19 @@ sidebar:
 
 {% include alert-development-section.md %}
 
-> Swish is a one-phase payment instrument supported by the major Swedish banks. In
-  the direct e-commerce scenario, Swedbank Pay receives the Swish registered
-  mobile number directly from the merchant UI. Swedbank Pay performs a payment
-  that the payer confirms using her Swish mobile app.
+{% include jumbotron.html body="Swish is a one-phase payment instrument
+supported by the major Swedish banks. In the direct e-commerce scenario,
+Swedbank Pay receives the Swish registered mobile number directly from the
+merchant UI. Swedbank Pay performs a payment that the payer confirms using her
+Swish mobile app." %}
 
 ## Introduction
 
+The Seamless View integration provide you with the Swish payment solution
+directly on your website. This gives the consumer a frictionless experience as
+we are handling the payment in the implemented `iframe` on your page (see
+example below).
```

Comment:
Perhaps this paragraph can be used in the jumbotron above instead?

Reply:
How about a soft intro  in the jumbotron and then use this text as a start to the section?

Reply:
Sure